### DISCUSSION TITLE
nvim colorschemes - only a select few colorschemes from `lua/plugins/extras/ui/colorschemes` are available to select..

### DISCUSSION THREAD

**Discussion by jontstaz** [Matt-FTW/dotfiles#16] [Q&A]
Hi,

I'm trying to work out why I can only select from a select few of the available colorschemes. Eg. I can use catppuccin fine but I can't use tokyonight or dracula despite them both existing in the `lua/plugins/extras/ui/colorschemes` folder. Why is that? How can I enable them all to be selectable?

Cheers,

**Comment by jontstaz** [comment]
Nevermind, I figured it out. Somehow I've only just tuned into the fact that many lazy packages are disabled and must be enabled via the `lazy extra` view... Love your nvim config btw, liking it much more than nvchad, astronvim, etc.

**Comment by Matt-FTW** [comment]
Hi Jontstaz!

Thats because I'm using the extras feature available in LazyVim. They are basically some plugin declarations that you can do in each file, and for each file that is inside the extras folder, you can enable and disable them in a custom menu. Its really cool and easy to try different plugins this way. That beign said, the colorschemes are extras.

To toggle the extras window hit `<space>lx` or just execute the `:LazyExtras` command. Then, head over to the colorschemes you want to install and restart neovim. Now you can change the colorschemes with `<leader>uC` or with the command `:Telescope colorscheme`.

Altough with this aproach you will have to change the colorscheme everytime you enter neovim because it chooses by default  catppuccin. To make your selected colorscheme persistent make sure to install the theme-loader extra. And after that, restart neovim and select your colorscheme. It will stay that way until you change it to another one.

Hope that works for you!

**Comment by Matt-FTW** [comment]
Opps just in time haha

thanks buddy <3. give a shot to lazyvim if you didnt, its the base of my config. I only created some custom extras
